.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: R2f6pCCIXKvoVhrNo1QeRIqAco414GHck 
```

Comment:
Not a good idea to push such credentials on public repositories. And according to [coveralls.io documentation](https://coveralls.zendesk.com/hc/en-us/articles/201769485-Ruby-Rails), the `.coveralls.yml` file is optional for public repositories 

> The option repo_token (found on your repository's page on Coveralls) is used to specify which project on Coveralls your project maps to. This is only needed for repos not using a CI and should be kept secret -- anyone could use it to submit coverage data on your repo's behalf. This shouldn't be a concern for private repos, however.